### PR TITLE
Feat: replaced pip with poetry

### PR DESCRIPTION
**Pull request by vitaliypopel**
Has been deleted requirements.txt, changed Dockerfile and added pyproject.toml. At all pip was replaced with poetry

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+sqlalchemy = "2.0.35"
+pytest = "8.3.3"
+pytest-asyncio = "0.24.0"
+pytest-env = "1.1.5"
+pytest-mock = "3.14.0"
+httpx = "0.27.2"
+celery = "5.4.0"
+redis = "5.2.0"
+trio = "0.27.0"
+
```
Comment (djeck1432) [inline pyproject.toml:31]: please, add `black` and `isort` to `tool.poetry.group.dev.dependencies`

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Please, attach also `poetry.lock` file

**Comment by vitaliypopel** [comment]
@djeck1432 done with it

**Review by djeck1432 — APPROVED** [review]
looks good to me

**Review by djeck1432 — APPROVED** [review]

**Pull request merged**